ktor-client/ktor-client-features/ktor-client-logging/common/src/io/ktor/client/features/logging/Logging.kt

```diff
@@ -80,6 +80,8 @@ public class Logging(
             logHeaders(request.headers.entries())
 
             logger.log("CONTENT HEADERS")
+            content.contentLength?.let { logger.logHeader(HttpHeaders.ContentLength, it.toString()) }
+            content.contentType?.let { logger.logHeader(HttpHeaders.ContentLength, it.toString()) }
```

Comment:
should not it be `HttpHeaders.ContentType`?

Reply:
Thanks for the notice, fixed